src/components/Signup/Signup.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import cookie from 'react-cookies';
+
+// Fake signup page, will later be replace by the sign up page
+export const Signup = () => {
+  const [username, setUsername] = useState('');
+
+  const changePath = () => {
+    cookie.save('username', username, { path: '/' });
+    window.location = '/token';
+  };
+
```

Comment:
I don't understand why you need to write the signup page. What you want to get from  signup page is just a username, so you can just use a dummy username for now. You should now waste your time to code the signup, and we need to review your signup at the same time

Reply:
Just want to make sure that I can get data from another component successfully, will not focus on this page anymore.